hikey_debian.mk

```diff
@@ -205,6 +205,7 @@ LINUX_CLEAN_COMMON_FLAGS += ARCH=arm64
 
 .PHONY: linux-clean
 linux-clean: linux-clean-common
+	rm -f $(ROOT)/linux-*
```

Comment:
That is a bit dangerous. Can you be slightly more specific (less wildcarding)?

Reply:
These are the files that need to go:
```
linux-4.9.0-optee-rpb_4.9.0-optee-rpb-1_amd64.changes
linux-4.9.0-optee-rpb_4.9.0-optee-rpb-1.debian.tar.gz
linux-4.9.0-optee-rpb_4.9.0-optee-rpb-1.dsc
linux-4.9.0-optee-rpb_4.9.0-optee-rpb.orig.tar.gz
linux-firmware-image-4.9.0-optee-rpb_4.9.0-optee-rpb-1_amd64.deb
linux-headers-4.9.0-optee-rpb_4.9.0-optee-rpb-1_amd64.deb
linux-image-4.9.0-optee-rpb_4.9.0-optee-rpb-1_amd64.deb
linux-libc-dev_4.9.0-optee-rpb-1_amd64.deb
```

so is below ok?
```
rm -f $(ROOT)/linux-*.changes
rm -f $(ROOT)/linux-*.tar.gz
rm -f $(ROOT)/linux-*.dsc
rm -f $(ROOT)/linux-*.deb
```

Reply:
Actually, what about `rm -f $(ROOT)/linux-*optee*.*`? Seems more efficient and safer too.

Reply:
Agreed.